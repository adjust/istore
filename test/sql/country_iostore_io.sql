BEGIN;
CREATE TABLE country_istore_io AS SELECT 'de=>1,de=>2'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1","en"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT ' "en"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=> "+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1" ,"de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1", "de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1","de"=>"2" '::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;

BEGIN;
CREATE TABLE country_istore_io AS SELECT 'en=>+1,de=>2 '::country_istore;
SELECT * FROM country_istore_io;
ROLLBACK;
