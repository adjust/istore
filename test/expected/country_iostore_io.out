BEGIN;
CREATE TABLE country_istore_io AS SELECT 'de=>1,de=>2'::country_istore;
SELECT * FROM country_istore_io;
 country_istore 
----------------
 "de"=>"3"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
 country_istore 
----------------
 "de"=>"3"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"de"=>"1","en"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"2","de"=>"1"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT ' "en"=>"+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=> "+1","de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1" ,"de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1", "de"=>"2"'::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT '"en"=>"+1","de"=>"2" '::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
BEGIN;
CREATE TABLE country_istore_io AS SELECT 'en=>+1,de=>2 '::country_istore;
SELECT * FROM country_istore_io;
   country_istore    
---------------------
 "zz"=>"1","de"=>"2"
(1 row)

ROLLBACK;
